Foundation/NSXMLParser.swift

```diff
@@ -233,9 +233,9 @@ internal func _NSXMLParserStartElementNs(ctx: _CFXMLInterface, localname: Unsafe
     let parser = ctx.parser
     let reportQNameURI = parser.shouldProcessNamespaces
     let reportNamespaces = parser.shouldReportNamespacePrefixes
-    let prefixLen = prefix == nil ? strlen(UnsafePointer<Int8>(prefix)) : 0
+    let prefixLen = prefix == nil ? UInt(strlen(UnsafePointer<Int8>(prefix))) : 0
```

Comment:
In swift/stdlib/public/SwiftShims/LibcShims.h swift_ssize_t is defined
```
#if defined(__linux__) && defined (__arm__)
typedef      int __swift_ssize_t;
#else
 typedef long int __swift_ssize_t;
#endif
```

 and later in swift/stdlib/public/stubs/LibcShims.cpp it is checked with a static assert to ensure that swift_ssize_t matches whatever the C ssize_t is.  Because of this, the size of ssize_t cannot be assumed to be a long int.

```
static_assert(std::is_same<ssize_t, swift::__swift_ssize_t>::value,
              "__swift_ssize_t is wrong");
```

I definitely don't think that this commit is the right way to do it.  This is another case of trying to get it to finish building to check linking.  I can revert this to what's existing for the purposes of this pull request.